.github/workflows/ci.yml

```diff
@@ -2,40 +2,42 @@ name: Python CI
 
 on:
   push:
-    branches: [main]
+    branches: 
+      - main
+      - 2u/project-theseus
+      - 2u/project-replatforming
```

Comment:
Can we remove this feature branch now from the CI run list?

Reply:
Fixed in 80a41b58431e2a6af8e9cd1bb700a61cb3a8cee0!